index.html

```diff
@@ -0,0 +1,263 @@
+                            <li> <i class="fa fa-circle"></i> Full Stack Dev</li>
+                            <li><i class="fa fa-circle"></i> 2015</li>
+                          </ul>
+                          </div>
+                        <p>
+                       Exploring the future of media in Facebook's first 
+                       Virtual Reality app; a place to
+                        discover and enjoy 360 photos and videos on Gear VR.</p>
+
+                        <div class="tags">
+                          <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                          </ul>
+                        </div>
+
+                        <a href="#" class="btn-two">See project</a>
+                      </li>
+                    </ul>
+                  </div>
+               </li>
+
+                <li class="work">            
+                  <div class="work-container">
+                    <ul>
+                      <li>
+                          <img src="image/portfolio4.png" alt="Several hands holding beer glasses">
+                      <li class="work-description">
+                        <h1><a href="">Uber Navigation</a></h1>
+                        <div class="work-info">
+                          <ul>
+                            <li class="intro">UBER</li>
+                            <li> <i class="fa fa-circle"></i> Lead Developer</li>
+                            <li><i class="fa fa-circle"></i> 2018</li>
+                          </ul>
+                        </div>
+                        <p>
+                        A smart assistant to make driving more safe, 
+                        efficient, and fun by unlocking 
+                        your most expensive computer: your car.</p>
+
+                        <div class="tags">
+                          <ul>
+                            <li>html</li>
+                            <li>Ruby on Rails</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                          </ul>
+                        </div>
+
+                        <a href="#" class="btn-one">See project</a>
+                      </li>
+                    </ul>
+                  </div>
+               </li>
+          </ul>
+  </section>
+
+<section class="section-about">
+<div class="container">
+<div class="about-grid">
+    <div class="text">
+      <div class="headline-text">
+          <h1 class="second-text">About Myself</h1>
+
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a
+             project you need coded, don’t hestiate to contact me.</p>
+
+            <a href="#" class="btn-connect"><h6>Let's connect</h6></a>
+          <ul class="medium">
```

Comment:
- [ ] Please refer to the design to fix `Let's connect` heading
![let](https://user-images.githubusercontent.com/55635977/126050474-e2a8bf66-e9dc-48c3-84cb-7cfde768e21d.png)